src/styles/radio.scss

```diff
@@ -5,11 +5,12 @@
       height: auto;
     }
   }
+}
 
-  @media only screen and (max-width: 500px) {
-    .ob-button-radio-container {
-      width: 100%;
-
+@container ob-form-container (max-width: 500px) {
+  .ob-button-radio-container {
+    width: 100%;
+    .ob-buttons-radio {
```

Comment:
this class if above `.ob-button-radio-container`, right?

Reply:
No.

Reply:
In the existing CSS `.ob-button-radio-container` is inside  `.ob-buttons-radio`.